### DISCUSSION TITLE
Plural forms

### DISCUSSION THREAD

**Discussion by milotype** [gotson/komga#1622] [Ideas]
@gotson 
Many languages (esspecialy slavic languages) need more than one plural form for strings with counts.
I'm not a programmer, so I don't know how to handle such strings separated with a bar character ' | '.

For instance the source string:
`No book | 1 book | {count} books`

I've translated it into Croatian as:
`Nema knjige | 1 knjiga | {count} knjige`
but Croatian needs two plural forms:
`Nema knjige | 1 knjiga | {count} knjige | {count} knjiga`

I don't know if adding more counts separated by a bar is possible. :-(
Please let me know if adding additional plural forms separated by a bar is possible/legit in this format.

Strings which I've found with this issue: https://hosted.weblate.org/search/komga/-/hr/?q=+%7C+&sort_by=-priority%2Cposition&checksum=

P.S.
This issue does not apply to the string `Series | Series`, as it's a simple singular/plural form without counts!

Cheers,
Milo

**Comment by gotson** [comment]
It's already supported for Polish, see https://github.com/gotson/komga/blob/kobo/komga-webui/src/i18n.ts#L23-L52 and https://kazupon.github.io/vue-i18n/guide/pluralization.html#custom-pluralization

PR welcome for other languages.

**Reply by milotype** [reply]
Thank you for your answer, but it unfortunately does not help me. I'm not a programmer, so I don't know how to add support for Croatian (or for other languages translated on [Weblate](https://hosted.weblate.org/projects/komga/#languages)).

As mentioned above please let me know if adding additional plural forms separated by a bar is possible/legit in this format and if that would solve the issue.

**Reply by gotson** [reply]
> As mentioned above please let me know if adding additional plural forms separated by a bar is possible/legit in this format and if that would solve the issue.

i already provided example on how Polish is handled. I you can't understand / implement then maybe someone will send a PR.

**Comment by gotson** [comment]
The new WebUI uses https://formatjs.github.io/ which supports plural forms.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.26.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.26.0))
